### PR TITLE
Improve account handling in Runtime

### PR DESCRIPTION
**Pull request by JanLukasGithub**
Before, accountName and config were saved, and to get the account you had to manually search for the right account in the config. Now the config and accountName are no longer saved in Runtime, because they're not used anywhere in the program, and instead the account is saved in Runtime

### CODE REVIEW

File: runtime/runtime.go [hunk 1]
```diff
@@ -269,8 +259,13 @@ func LoadEnvVariables(defaultAc AccountEntry) AccountEntry {
 // used for testing.
 func NewTestRuntime() (*Runtime, error) {
 	rt := &Runtime{
-		accountName: "test",
-		client:      nil,
+		account: AccountEntry{
+			Name:   "test",
+			UserID: "testId",
+			Token:  "testToken",
+			URL:    "testURL",
```
Comment (JanLukasGithub) [inline runtime/runtime.go:266]: Use a URL like "testURL.example.com"

**Comment by bkircher** [comment]
Can you rebase manually please?

**Comment by JanLukasGithub** [comment]
> Can you rebase manually please?

Done I think

**Comment by bkircher** [comment]
Yep, Thanks!

**Pull request merged**